tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm5_1_1

```diff
@@ -191,7 +184,7 @@ CMD ["/bin/bash"]
 #Build manylinux2014 docker image end
 
 ARG PYTHON_VERSION=3.7
-ARG TORCH_VERSION=1.11.0
+ARG TORCH_VERSION=1.10.0
```

Comment:
Why do we change default `TORCH_VERSION` to `1.10.0`? Shall we keep it at `1.11.0`?

Reply:
Yes, I think it should keep 1.11.0